libs/parentParsers.py

```diff
@@ -89,7 +89,11 @@
     '--cleanup-clusters',
     default=True,
     help='Cleanup any non-error state clusters upon test completion')
-
+clusterParser.add_argument(
+    '--only-delete-clusters',
+    dest='only_delete_clusters',
+    action='store_true',
+    help='Delete clusters found on folder specified by --path')
```

Comment:
@morenod Can we add a bit more to the help saying that it will exit after deleting. ie its not meant to be run at the same time as a test.